installation/README.md

```diff
@@ -1,3 +1,95 @@
 # Kyma Installer
 
-The Kyma installer is a proprietary solution based on the Kubernetes operator. See the [Local Kyma installation](../docs/kyma/docs/031-gs-local-installation.md) document for basic information on how to use the installer and run Kyma locally.  
+The Kyma installer is a proprietary solution based on the Kubernetes operator. See the [Local Kyma installation](../docs/kyma/docs/031-gs-local-installation.md) document for basic information on how to use the installer and run Kyma locally.
+
+## Cluster installation
+
+> **NOTE:** The remainder of this section assumes you have access to a running cluster.
+
+To install Kyma, you will need the following data:
+
+- Domain name
+- Wildcard TLS certificate for your cluster domain (can be generated with Let's Encrypt)
+- Certificate for Remote Environments
+- IP address for Kyma Ingress
+- IP address for Remote Environments Ingress
+
+### 0. Prerequisites
+
+Kubernetes API Server is configured in the following manner:
+
+```
+"apiServerConfig": {
+    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota,PodPreset",
+    "--runtime-config": "batch/v2alpha1=true,settings.k8s.io/v1alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
+    "--cors-allowed-origins": ".*",
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--oidc-issuer-url": "https://dex.__DOMAIN_NAME__",
```

Comment:
Explain that __DOMAIN_NAME__ is a placeholder thus this entire thing is a template.